### PR TITLE
Update to Cloud plan characteristics

### PR DESCRIPTION
**Pull request by danjohansenconsulting**
The Free, Team, and Business plans each allow for 50 users.

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@danjohansenconsulting 

This is not in alignment with the information on https://www.cypress.io/pricing

image

File: docs/guides/cloud/account-management/billing-and-usage.mdx [hunk 1]
```diff
@@ -65,10 +65,6 @@ OSS plan.
      src="/img/guides/cloud/organizations/oss-plan-3-billing.jpg"
      alt="Upgrade Plan"
    />
-   <DocsImage
```
Comment (jaffrepaul) [inline docs/guides/cloud/account-management/billing-and-usage.mdx:68]: If this is no longer part of the flow, can you delete the image from the directory as well?

File: docs/guides/references/changelog.mdx [hunk 2]
```diff
@@ -2,6 +2,14 @@
 title: Changelog
 ---
```
Comment (jaffrepaul) [inline docs/guides/references/changelog.mdx:4]: This commit shouldn't be necessary. The changelog page is a landing page of sorts of the the OSS app releases, which is still at `13.3.3`.

File: docs/partials/_cloud_free_plan.mdx [hunk 3]
```diff
@@ -1,4 +1,4 @@
 To get started with Cypress Cloud,
 [sign up](https://cloud.cypress.io/signup) for our free plan.
 
-Free plans allow **3** users and **500** monthly test results, while exceeding this limit requires upgrading the plan. Please see our [pricing page](https://www.cypress.io/pricing) for more details.
+Free plans allow **50** users and **500** monthly test results, while exceeding this limit requires upgrading the plan. Please see our [pricing page](https://www.cypress.io/pricing) for more details.
```
Comment (jaffrepaul) [inline docs/partials/_cloud_free_plan.mdx:4]: Cool. Just confirming there are no Team and Business plans references that need any editing while we're in here?

**Review by jaffrepaul — CHANGES_REQUESTED** [review]
Added a couple of small changes/questions.

File: docs/partials/_cloud_free_plan.mdx (hunk 3 above)
Comment (danjohansenconsulting) [reply docs/partials/_cloud_free_plan.mdx:4]: Correct, I don't see anything for Team or Business

**Comment by jaffrepaul** [comment]
closing in favor of https://github.com/cypress-io/cypress-documentation/pull/5548

**Pull request closed without merging**